src/rust-util/pelikan-sys/Cargo.toml

```diff
@@ -22,10 +24,13 @@ slab = []
 time = []
 util = [ "time" ]
 
-default = [ "core", "time", "util" ]
+default = [ 
+    "core", "time", "util",
```

Comment:
Any particular reason for this formatting? I'd prefer one line, or line per feature here.

Reply:
This was left over from when I added some other default features temporarily. I changed it to be all on one line.